subprojects/core/src/main/java/org/gradle/cache/internal/btree/BlockPointer.java

```diff
@@ -15,14 +15,29 @@
  */
 package org.gradle.cache.internal.btree;
 
+import com.google.common.primitives.Longs;
+
 public class BlockPointer implements Comparable<BlockPointer> {
```

Comment:
That is on my bucket list, possibly even removing the need for `List<IndexEntry>` (which is bad for cache locality).

For now I just didn't want to change too much code at a time.